partcad/src/partcad/project.py

```
@@ -1385,6 +1388,20 @@ async def render_async(
 
+                if (
+                    "brep" in shape_render
+                    and shape_render["brep"] is not None
+                    and not isinstance(shape_render["brep"], str)
+                    and shape.kind in shape_render.get("brep", {}).get("exclude", [])
+                ):
+                    render_brep = False
+                elif format is None and "brep" in shape_render:
+                    render_brep = True
+                elif not format is None and format == "brep":
+                    render_brep = True
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)